api/src/utils/encryption/index.test.ts

```diff
@@ -1,4 +1,4 @@
-import { createChallenge, decrypt, encrypt, getHexEncodedKey, randomSecretKey, signChallenge, verifiyChallenge } from '.';
+import { createNonce, decrypt, encrypt, getHexEncodedKey, randomSecretKey, signNonce as signNonce, verifySignedNonce } from '.';
```

Comment:
`signNonce as signNonce` is probably redundant

Reply:
yes weird refactoring issue but fixed it